bitwarden/values.yml

```diff
@@ -0,0 +1,19 @@
+replicaCount: 1
+
+image:
+  repository: bitwardenrs/server-mysql
+  tag: latest
```

Comment:
Please pin this

Reply:
Just use whatever bitwarden version is on latest right now

Reply:
Pinned